src/features/pathway/components/PathwayPanel.jsx

```
@@ -0,0 +1,1975 @@
+          minHeight: 0,
+          flex: 1,
+          overflowY: 'auto',
+          display: 'flex',
+          flexDirection: 'column',
+          gap: 14,
+          paddingRight: 2,
+        }}
+      >
+
+      {panelError ? (
+        <Alert
```

ESLint

[error] Insert `··`

(prettier/prettier)